### PR TITLE
LPS-154215 Remove LIFERAY_LIB_GLOBAL_SHARED_DIR since it is no longer needed

### PR DESCRIPTION
**Pull request by julschong**
Hi Tina, 

per our discussion, this pull removes the global shared dir property in order to eventually remove the _getLibDir() in portal-impl/PropsUtil

### CODE REVIEW

**Comment by julschong** [comment]
@catarineaguiar Hi Catarine, could you please help me scan this test result? looks pretty good :D
Thank you!

**Comment by julschong** [comment]
resolved  conflict for 

> portal-kernel/bnd.bnd

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
@catarineaguiar @tinatian 
conflict only consists of bnd.bnd so previous test result should be able to be used to scan
Thank you!

**Comment by catarineaguiar** [comment]
@julschong no regressions were found.

**Comment by tinatian** [comment]
@julschong Code conflicts, please rebase.

Thanks.
Tina.

**Comment by julschong** [comment]
rebased and resolved conflict for:

**Comment by tinatian** [comment]
@julschong You need to add the property you removed to VerifyProperties, please check with Jeff if you have any question about it.

Tina.

**Comment by julschong** [comment]
Added the last commit to put `liferay.lib.global.shared.dir` into `_OBSOLETE_PORTAL_KEYS` in VerifyProperties

**Comment by julschong** [comment]
rebased and resolve confliect for:

> portal-kernel/src/com/liferay/portal/kernel/util/packageinfo

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina

File: portal-impl/src/com/liferay/portal/events/StartupAction.java [hunk 1]
```diff
@@ -67,28 +62,6 @@ public void run(String[] ids) throws ActionException {
 
 	protected void doRun(String[] ids) throws Exception {
 
-		// Check Tomcat's lib/ext directory
```
Comment (shuyangzhou) [inline portal-impl/src/com/liferay/portal/events/StartupAction.java:70]: @julschong @tinatian This block of code was just added last month, you can not just delete it. This complain must be preserverd, Brian asked for it. You can remove the property, but still need to calculate out the location in order to scan for complain. Please fix and resend.

**Comment by shuyangzhou** [comment]
ci:close

See above comment.

File: portal-impl/src/com/liferay/portal/events/StartupAction.java (hunk 1 above)
Comment (julschong) [reply portal-impl/src/com/liferay/portal/events/StartupAction.java:70]: @tinatian Hi Tina, I will inline _getLibDir into `StartupAction` and still remove the globalSharedLib prop

**Pull request closed without merging**